nupm/install.nu

```diff
@@ -153,7 +153,7 @@ def download-pkg [
     let pkg_dir = if $pkg.path == null {
         $env.PWD | path join $clone_dir
     } else {
-        $env.PWD | path join $clone_dir ($pkg.path | path dirname)
```

Comment:
this previously failed on `nupm install nu-clippy --registry nupm_test`

Reply:
Sorry to hear that, @cptpiepmatz @fdncred are you able to take a look at @Rydwxz's PR?